agent/consul/server_test.go

```diff
@@ -767,10 +767,10 @@ func TestServer_RevokeLeadershipIdempotent(t *testing.T) {
 	dir1, s1 := testServer(t)
 	defer os.RemoveAll(dir1)
 	defer s1.Shutdown()
-	
+
 	testrpc.WaitForLeader(t, s1.RPC, "dc1")
 
-	err:= s1.revokeLeadership()
+	err := s1.revokeLeadership()
```

Comment:
**Note:** The seemingly spurious changes in this file result from changes applied by `make format`.